docs/articles/documentation/using-testcafe/programming-interface/testcafe.md

```diff
@@ -100,6 +101,39 @@ createTestCafe('localhost', 1337, 1338)
     });
 ```
 
+### createLiveModeRunner
+
+Creates a [test runner](livemoderunner.md) that runs TestCafe in [live mode](../common-concepts/live-mode.md). In this mode, TestCafe watches for changes you make in the test files and all files referenced in them (like page objects or helper modules). These changes immediately restart the tests so that you can see the effect.
+
+```text
+createLiveModeRunner() → LiveModeRunner
+```
+
+[LiveModeRunner](livemoderunner.md) is a [Runner](runner.md) descendant and provides the same API (with certain [limitations](livemoderunner.md#limitations)).
+
+**Example**
+
+```js
+const createTestCafe = require('testcafe');
+let testcafe         = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(tc => {
+        testcafe         = tc;
+        const liveRunner = testcafe.createLiveModeRunner();
+
+        return liveRunner
+            .src('tests/test.js')
+            .browsers('chrome')
+            .run();
+    })
+    .then(() => {
+        testcafe.close();
+    });
+```
+
+> Important! You cannot create multiple live mode runners for the same TestCafe server instance. This is because a TestCafe server can handle only one watcher that tracks changes to test files.
+
 ### close
 
 Stops the TestCafe server. Forcibly closes all connections and pending test runs immediately.
```

Comment:
Stops the TestCafe server. Forcibly closes all connections and pending test runs.